proto/provenance/name/v1/tx.proto

```diff
@@ -46,3 +49,18 @@ message MsgDeleteNameRequest {
 
 // MsgDeleteNameResponse defines the Msg/DeleteName response type.
 message MsgDeleteNameResponse {}
+
+// MsgCreateRootNameProposal defines an sdk.Msg type to create a new root name
+// that is controlled by a given owner and optionally restricted to the owner
+// for the sole creation of sub names.
+message MsgCreateRootNameRequest {
```

Comment:
Gotta have a new message primarily so that the authority can be provided and checked. Relatedly, the "fromAddress" here should be "authority" to match other gov prop messages as well as what it's checked against in the keeper.